src/main/java/com/minecolonies/colony/buildings/BuildingBuilder.java

```diff
@@ -13,44 +13,85 @@
  */
 public class BuildingBuilder extends AbstractBuildingWorker
 {
+    /**
+     * The maximum upgrade of the building.
+     */
+    private final static int MAX_BUILDING_LEVEL = 2;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)